timemachine/fe/free_energy.py

```diff
@@ -351,7 +353,9 @@ def sample(
 
     Returns
     -------
-    xs, boxes: np.arrays with .shape[0] = md_params.n_frames
+    xs, boxes: NDArray with .shape[0] = md_params.n_frames
+
+    final_velocities: NDArray with shape (N,)
```

Comment:
This docstring is technically incorrect, I think this function returns a single tuple with 3 components, (xs, boxes, final_velocities), as opposed to 2 components ((xs, boxes), final_velocities).

Reply:
Ah yeah, the grouping of `xs, boxes` wasn't meant to indicate they were in a separate tuple. Will split lines to clarify.

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/1142/commits/2fe7c2fb85aac0405b8cae193280760ed8e209a8